omniport/omniport/middleware/drf_auth.py

```diff
@@ -53,6 +56,12 @@ def __call__(self, request):
 
         request.user = SimpleLazyObject(lambda: get_user_jwt(request))
 
+        # Provide the user context to any exception raised by sentry
+        if 'sentry' in _CONF.integrations:
+            with configure_scope() as scope:
+                if request.user:
+                    scope.user = {"username": request.user.username}
```

Comment:
Add some more info, like roles or contact details, just in case.

Reply:
@dhruvkb I am surprised you overlooked the `""` in `"username"` :open_mouth:

Reply:
I have more faith in you than I do in myself. I firmly believed you would catch that.